packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -167,6 +167,7 @@ class OnsetPicker extends React.Component {
     const onsetObject = {
       precision: this.state.precision,
       onsetDate: this.state.onsetDate ? this.state.onsetDate.format(DATE_FORMAT) : undefined,
+      onsetDateMetadata: this.state.onsetDateMetadata,
```

Comment:
Now that we have `onsetDateMetadata` added to the onsetObject returned in the callback, can we update the [onsetOnChange prop description](https://github.com/cerner/terra-clinical/blob/terra-clinical-onset-picker%404.30.0/packages/terra-clinical-onset-picker/src/OnsetPicker.jsx#L69-L70) to also include onsetDateMetadata so that consumers are in the know?

Reply:
Updated: https://github.com/cerner/terra-clinical/pull/784/commits/3915af23b50e782d5fe8aa0286d47d96622cf199